src/index.html

```diff
@@ -1,13 +1,435 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>THE MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/logo.svg"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+
+            <a href="/"class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.svg"
+                alt="MET"
+              >
+            </a>
+            <div class="header__wrapper">
+              <a href="tel:+1 212-535-7710"class="icon icon--phone header__wrapper__call">
+              </a>
```

Comment:
these numbers are too close to page border, try to move them down a bit:
![image](https://user-images.githubusercontent.com/37161315/170512365-e678a7f6-4598-4baa-9549-2eb08b9b809c.png)